recipes/imgui-sfml/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, replace_in_file, rm, rmdir, mkdir
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class ImGuiSFMLConan(ConanFile):
+    name = "imgui-sfml"
+    description = "ImGui-SFML integrates Dear ImGui with SFML, enabling easy creation of GUIs for SFML applications."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/SFML/imgui-sfml"
+    topics = ("gui", "graphical", "sfml", "imgui")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_sfml": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_sfml": True
```

Comment:
Does `with_sfml=False` even do anything sensible? I think setting `IMGUI_SFML_FIND_SFML=OFF` simply breaks the build.

Reply:
The IMGUI_SFML_FIND_SFML variable allows the user to decide whether CMake should automatically find and configure SFML using find_package, or if the user will manually manage SFML integration. Maybe you're right and it's not a necessary option.